sharding/collation.go

```diff
@@ -45,3 +54,83 @@ func (c *Collation) AddTransaction(tx *types.Transaction) {
 	// TODO: Include blob serialization instead.
 	c.transactions = append(c.transactions, tx)
 }
+
+// CreateRawBlobs creates raw blobs from transactions.
+func (c Collation) CreateRawBlobs() ([]*utils.RawBlob, error) {
+
+	// It does not skip evm execution by default
+	blobs := make([]*utils.RawBlob, len(c.transactions))
+	for i := 0; i < len(c.transactions); i++ {
+
+		err := error(nil)
+		blobs[i], err = utils.NewRawBlob(c.transactions[i], false)
+
+		if err != nil {
+			return nil, fmt.Errorf("Creation of raw blobs from transactions failed %v", err)
```

Comment:
can we add a colon at the end? and for the rest of the code

Reply:
I am not sure where exactly you are referring to